pom.xml

```diff
@@ -38,6 +38,31 @@
         <module>client</module>
     </modules>
 
+    <dependencies>
```

Comment:
The correct versions for these dependencies can be found at:
https://github.com/diffblue/platform/blob/develop/docs/user/topics/executing-diffblue-tests.md#maven-projects

Reply:
Changed versions to 1.9.0 and 1.6.6